build/package/Installer.MSI.targets

```diff
@@ -188,7 +192,23 @@
       
       <GenerateMSBuildExtensionsSWR MSBuildExtensionsLayoutDirectory="$(MSBuildExtensionsLayoutDirectory)"
                                     OutputFile="$(SdkMSBuildExtensionsSwrFile)"/>
+    </Target>
+
+    <Target Name="GenerateSdkSymbolsNupkg"
+            DependsOnTargets="Init;Layout"
+            Condition=" '$(OS)' == 'Windows_NT' And '$(SeparateSymbolsArchive)' != 'false' "
```

Comment:
Quick check, intention is to not generate a separate symbols package for non-windows OS's?

Reply:
@mmitche 
No, the intention is to generate a separate symbols package for all RIDs. This code will be changed - it was a faulty first go at it.